tasks/rh-sign-image/tests/test-rh-sign-image-ir-failure.yaml

```diff
@@ -46,7 +46,6 @@ spec:
               cat > "$(workspaces.data.path)/data.json" << EOF
               {
                 "sign": {
-                  "request": "hacbs-signing-pipeline",
```

Comment:
Just to be clear, this was really redundant in the test even before your change, right?

Reply:
ok, cool